### PR TITLE
Skype client

### PR DESCRIPTION
**Pull request by alexlarsson**

### CODE REVIEW

**Comment by alexlarsson** [comment]
There is a bunch of udev binaries in this, maybe they should be cleaned up by the shared module.

**Comment by alexlarsson** [comment]
Build: https://flathub.org/builds/#/builders/1/builds/460

File: com.skype.Client.json [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+                    "sha256": "1912b91803ab09a5eed34d364bf09fe3a2a9c96751fde03a4e0cfa51a04d784c"
+                }
+            ]
+        },
+        {
+            "name": "libsecret",
+            "config-opts": ["--disable-gtk-doc",
+                            "--enable-introspection=no",
+                            "--disable-man",
+                            "--with-xinput=xfree"],
```
Comment (cosimoc) [inline com.skype.Client.json]: I don't think `libsecret` has this option.

**Comment by TingPing** [comment]
Repository has been created: https://github.com/flathub/com.skype.Client

**Pull request closed without merging**